src/StudioCore/Assets/Paramdex/ER/Upgrader/massedit.txt

```diff
@@ -48,4 +48,529 @@ param ToughnessParam: added: unk1: = 1
+11210015L
+1.12.1 - (CharaInitParam) Set unknown_0x51 to Vanilla v1.12.1 values
+param CharaInitParam: modified && !added: unknown_0x51: = vanillafield unknown_0x51
+11210015L
+1.12.1 - (CharaInitParam) Set pad8 to Vanilla v1.12.1 values
+param CharaInitParam: modified && !added: pad8: = vanillafield pad8
+11210015L
+1.12.1 - (BulletParam) Set unknown_0x98_3 to Vanilla v1.12.1 values
+param BulletParam: modified && !added: unknown_0x98_3: = vanillafield unknown_0x98_3
+11210015L
+1.12.1 - (BulletParam) Set pad to Vanilla v1.12.1 values
+param BulletParam: modified && !added: pad: = vanillafield pad
+11210015L
+1.12.1 - (BulletParam) Set unknown_0xc3_5 to Vanilla v1.12.1 values
+param BulletParam: modified && !added: unknown_0xc3_5: = vanillafield unknown_0xc3_5
+11210015L
+1.12.1 - (BonfireWarpParam) Set dispMask02 to Vanilla v1.12.1 values
+param BonfireWarpParam: modified && !added: dispMask02: = vanillafield dispMask02
+11210015L
+1.12.1 - (BonfireWarpParam) Set pad1 to Vanilla v1.12.1 values
+param BonfireWarpParam: modified && !added: pad1: = vanillafield pad1
+11210015L
+1.12.1 - (AtkParam) Set subCategory3 to Vanilla v1.12.1 values
+param AtkParam: modified && !added: subCategory3: = vanillafield subCategory3
+11210015L
+1.12.1 - (AtkParam) Set subCategory4 to Vanilla v1.12.1 values
+param AtkParam: modified && !added: subCategory4: = vanillafield subCategory4
+11210015L
+1.12.1 - (AtkParam) Set pad7 to Vanilla v1.12.1 values
+param AtkParam: modified && !added: pad7: = vanillafield pad7
+11210015L
+1.12.1 - (AssetGeometryParam) Set unknown_0x3b_7 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: unknown_0x3b_7: = vanillafield unknown_0x3b_7
+11210015L
+1.12.1 - (AssetGeometryParam) Set unknown_0x120 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: unknown_0x120: = vanillafield unknown_0x120
+11210015L
+1.12.1 - (AssetGeometryParam) Set unknown_0x121 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: unknown_0x121: = vanillafield unknown_0x121
+11210015L
+1.12.1 - (AssetGeometryParam) Set unknown_0x122 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: unknown_0x122: = vanillafield unknown_0x122
+11210015L
+1.12.1 - (AssetGeometryParam) Set unknown_0x123 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: unknown_0x123: = vanillafield unknown_0x123
+11210015L
+1.12.1 - (AssetGeometryParam) Set unknown_0x124 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: unknown_0x124: = vanillafield unknown_0x124
+11210015L
+1.12.1 - (AssetGeometryParam) Set unknown_0x125 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: unknown_0x125: = vanillafield unknown_0x125
+11210015L
+1.12.1 - (AssetGeometryParam) Set Reserve_0 to Vanilla v1.12.1 values
+param AssetGeometryParam: modified && !added: Reserve_0: = vanillafield Reserve_0
```

Comment:
Nice to see we don't have a heap of fields being actively moved.
Are there any additions where we should modify modified && added to = 1 for example, like damagerate fields that are otherwise going to be initialised as 0?
I don't see any new refs like we had for finaldamagerate, but same story.